tests/test_file.py

```diff
@@ -19,67 +19,74 @@ def test_env(self):
         os.environ["ISITWEEKENDYET"] = "0"
         self.assertFalse(psf.env_get("ISITWEEKENDYET", boolean=True))
 
-    def test_get_sentinel_scene_from_dir(self):
+    def test_get_sentinel_scene_from_dir_path_testfiles(self):
         with psf.get_sentinel_scene_from_dir(path_testfiles) as (full_path, ident):
             self.assertEqual("S1M_hello_from_inside", ident)
-
+    def test_get_sentinel_scene_from_dir_str_path(self):
         with psf.get_sentinel_scene_from_dir(str_path) as (full_path, ident):
             self.assertEqual("S1M_hello_from_inside", ident)
-
+    def test_get_sentinel_scene_from_dir_joinpath(self):
         with psf.get_sentinel_scene_from_dir(path_testfiles.joinpath("another_scene")) as (full_path, ident):
             self.assertEqual("S2__IN_FOLDER", ident)
 
-    def test_get_polarization_from_s1_filename(self):
+    def test_get_polarization_from_s1_filename_1SDH(self):
         self.assertEqual(
             psf.get_polarization_from_s1_filename(
                 "MMM_BB_TTTR_1SDH_YYYYMMDDTHHMMSS_YYYYMMDDTHHMMSS_OOOOOO_DDDDDD_CCCC.SAFE.zip"
             ),
             "HH",
         )
+    def test_get_polarization_from_s1_filename_1SSH(self):
         self.assertEqual(
             psf.get_polarization_from_s1_filename(
                 "MMM_BB_TTTR_1SSH_YYYYMMDDTHHMMSS_YYYYMMDDTHHMMSS_OOOOOO_DDDDDD_CCCC.SAFE.zip"
             ),
             "HH",
         )
+    def test_get_polarization_from_s1_filename_2SSV(self):
         self.assertEqual(
             psf.get_polarization_from_s1_filename(
                 "MMM_BB_TTTR_2SSV_YYYYMMDDTHHMMSS_YYYYMMDDTHHMMSS_OOOOOO_DDDDDD_CCCC.SAFE.zip"
             ),
             "VV",
         )
+    def test_get_polarization_from_s1_filename_1SDV(self):
         self.assertEqual(
             psf.get_polarization_from_s1_filename(
                 "MMM_BB_TTTR_1SDV_YYYYMMDDTHHMMSS_YYYYMMDDTHHMMSS_OOOOOO_DDDDDD_CCCC.SAFE.zip", True,
             ),
             "VV,VH",
         )
 
-    def test_get_ts_from_sentinel_filename(self):
+    def test_get_ts_from_sentinel_filename_S3M_TTT(self):

```

Comment:
Check what the real test are in here. It is either S1, S2 or S3 and could also be an enddate.